examples/integration-gin/main.go

```diff
@@ -9,7 +9,12 @@ import (
 
 func main() {
 	engine := gin.Default()
-	engine.HTMLRender = gintemplrenderer.Default
+	engine.LoadHTMLFiles("./home.html")
+
+	//engine.HTMLRender = gintemplrenderer.Default
+
+	ginHtmlRenderer := engine.HTMLRender
+	engine.HTMLRender = &gintemplrenderer.HTMLTemplRenderer{FallbackHtmlRenderer: ginHtmlRenderer}
```

Comment:
This doesn't feel right. I don't see why the variable exists.

Reply:
the variable can be removed indeed, it was to make the code a little more readable so that user can understand that here we are setting a fallback to gin default html renderer in case we do not pass a templ